website/docs/cli/cloud/settings.mdx

```diff
@@ -84,14 +84,16 @@ The `cloud` block supports the following configuration arguments:
 
 ### Environment Variables
 
-Alternatively, the `cloud` block can be configured with the following environment variables:
+Alternatively, you can configure the `cloud` backend non-interactively using environment variables, such as when configuring Terraform in a CI pipeline. The `cloud` block can be configured with the following environment variables:
+
+-> **Note:** These variables are only read if the corresponding attribute is omitted from your configuration. They behave as fallbacks rather than overriding your configuration. Furthermore, while all attributes can be omitted, an empty `cloud` block must still be present in your configuration.
 
 - `TF_ORGANIZATION` - The name of the organization. Serves as a fallback for `organization`
     in the cloud configuration. If both are specified, the configuration takes precedence. 
 
 - `TF_HOSTNAME` - The hostname of a Terraform Enterprise installation. Serves as a fallback if `hostname` is not specified in the cloud configuration. If both are specified, the configuration takes precendence. 
 
-- `TF_WORKSPACE` - The workspace to select from the available set. The `cloud` block interprets this variable as the `name` value of the `workspaces` attribute allowing you to omit this attribute completely. It does not alter the existing behavior of how Terraform interprets this variable. [Read more about TF_WORKSPACE](https://www.terraform.io/cli/config/environment-variables#tf_workspace)
+- `TF_WORKSPACE` - The workspace to select from the available set. The `cloud` block interprets this variable as the `name` value of the `workspaces` attribute. The cloud backend will only read this variable if `workspaces` is completely omitted from your configuration. It also does not alter the existing behavior of how Terraform interprets this variable. [Read more about TF_WORKSPACE](https://www.terraform.io/cli/config/environment-variables#tf_workspace)
```

Comment:
A suggestion and then the last sentence confuses me a bit. I wonder if we should say something more like "Terraform will continue to use this variable to select the workspace locally. Refer to [Environment Variables](link) for details." 

I just think we should tell users briefly what the behavior IS - in case it affects them wanting to use this for the `cloud` block?

```suggestion
- `TF_WORKSPACE` - The workspace to select from the available set. The `cloud` block interprets this variable as the `name` value of the `workspaces` attribute. The `cloud` block only reads this variable if `workspaces` is not set in your configuration file. It also does not alter the existing behavior of how Terraform interprets this variable. [Read more about TF_WORKSPACE](https://www.terraform.io/cli/config/environment-variables#tf_workspace)
```